planning/behavior_path_planner/include/behavior_path_planner/scene_module/start_planner/start_planner_module.hpp

```diff
@@ -104,6 +104,19 @@ class StartPlannerModule : public SceneModuleInterface
   {
   }
 
+  // set is_simulataneously_executable_ as false when backward driving.
+  // keep initial value to return it after finishing backward driving.
+  bool initial_is_simultaneously_executable_as_approved_module_;
```

Comment:
[imo] `initial_is_*_ is` little bit a unreadable.
`initial_value_simultaneously_executable_as_approved_module_` ?

Reply:
it' better, fixed in 9c215eaa6aa72d5547f40e324cdd5a75cd001f28